### DISCUSSION TITLE
Как настроить подсветку ключевых слов в Vivado

### DISCUSSION THREAD

**Discussion by rainfault** [MPSU/exAPS#27] [Вопросы и ответы] [answered]
Как настроить подсветку ключевых слов (reg, wire, etc.) в Vivado 2019.2?

По дефолту, подсветка отсутствует, вместо этого кейворды выделяются курсивом.

![image](https://user-images.githubusercontent.com/99883153/196029244-484ecb7b-3813-486a-b2f2-79a1764a981a.png)

**Comment by gr33nka** [comment, marked as answer]
https://support.xilinx.com/s/article/57719?language=en_US

`Tools` -> `Options...` -> `General`
Выбираешь `Text Editor` и там вроде где-то должно настраиваться. Под рукой нет Vivado, думаю там должно быть интуитивно

**Reply by rainfault** [reply]
Если цвет подсветки стоит, но она не появилась, просто нажмите ОК
![image](https://user-images.githubusercontent.com/99883153/196030442-56503a96-0d24-41f0-a6db-06e30d50153d.png)
